session-replacement/src/main/java/com/amadeus/session/repository/redis/RedisConfiguration.java

```diff
@@ -138,6 +170,15 @@ public String getPoolSize() {
   public RedisConfiguration(SessionConfiguration conf) {
     readConfigurationString(conf.getProviderConfiguration());
     serverAddress(conf);
+    if (password == null) {
+      password = conf.getAttribute(REDIS_PASSWORD, null);
+    }
+    if (useSSL == false) {
+      useSSL = stringToBool(conf.getAttribute(REDIS_SSL, "false"));
+    }
+    if (tls == null ){
+        tls = tlsStringToArray(conf.getAttribute(REDIS_TLS, "TLSv1, TLSv1.1, TLSv1.2"));
```

Comment:
Indentation

Reply:
sure will fix it. And just noticed I missed the square brackets in the default value.  It should be like this - `conf.getAttribute(REDIS_TLS, "[TLSv1, TLSv1.1, TLSv1.2]")`